### PR TITLE
Updated resource page documentation to include new feature

### PR DESCRIPTION
**Pull request by janewang-datadog**
### What does this PR do?

Updated resource page docs to include Dependency Map Navigator feature

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/services/resource_page.md [hunk 1]
```diff
@@ -61,21 +61,24 @@ Use the top right percentile selectors to zoom into a given percentile, or hover
...
 
 The highlight color of the node indicates the service's [monitor status][5]. If a service has more than one configured monitor, the status of the most severe monitor is shown.
 
-{{<img src="tracing/visualization/resource/resource_dependency_map_hover.mp4" video="true" alt="hovering and clicking a resource dependency map node" style="width:100%;">}}
+{{< img src="tracing/visualization/resource/dependency-navigator.mp4" video="true" alt="hovering and clicking on the list" style="width:100%;">}}
+
+A service has load amplification if it's receiving more than 100% of the requests received by the selected resource upstream. Services with call paths highlighted in orange have load amplification, and the amplification (multiplier) is shown in the list on the panel. The amplification is calculated based on the requests received by the resource (shown on the map in the image below), and the requests received by the service (shown in the service node on the map). By clicking on a service in the list, you can see the spans contributing to the amplification.
```
Comment (buraizu) [inline content/en/tracing/services/resource_page.md:78]:
```suggestion
A service has load amplification if it's receiving more than 100% of the requests received by the selected resource upstream. Services with call paths highlighted in orange have load amplification, and the amplification multiplier is shown in the list on the panel. The amplification is calculated based on the requests received by the resource (shown highlighted on the map in the image below), and the requests received by its downstream service nodes (also shown on the map). By clicking on a service in the list, you can see the spans contributing to the amplification.
```

Comment (buraizu) [reply content/en/tracing/services/resource_page.md:78]: Minor update to clarify the image, but let me know if any of this wording is incorrect

**Review by buraizu — CHANGES_REQUESTED** [review]
Nice update, thanks for fixing those typos and other issues! The suggestion in this review add a bit more detail to some of the alt text, and also update an image which is not appearing in preview.

File: content/en/tracing/services/resource_page.md [hunk 2]
```diff
@@ -18,13 +18,13 @@ further_reading:
   text: "Understand how to read a Datadog Trace"
 ---
 
-{{< img src="tracing/visualization/resource/ressource.png" alt="Ressource" >}}
+{{< img src="tracing/visualization/resource/resource-page.png" alt="resource" >}}
```
[the remark this answers is not in the document]
Comment (janewang-datadog) [reply content/en/tracing/services/resource_page.md:21]: This isn't a APM service page - it's a resource page 
We can say "APM resource page" instead?

File: content/en/tracing/services/resource_page.md (hunk 1 above)
Comment (janewang-datadog) [reply content/en/tracing/services/resource_page.md:80]: "and the requests received by the downstream service (shown inside the downstream service node on the map)"

Comment (buraizu) [reply content/en/tracing/services/resource_page.md:80]: Sounds good to me, updated the suggestion

File: content/en/tracing/services/resource_page.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (buraizu) [reply content/en/tracing/services/resource_page.md:21]: Thanks for clarifying, the suggestion was based off of the `Services` tab being selected, but I'll update back to resource page

**Review by buraizu — APPROVED** [review]
LGTM

**Pull request merged**